pkg/urbit/vere/pier.c

```diff
@@ -935,6 +935,48 @@ _pier_on_lord_bail(void* vod_p)
   u3_pier_bail(pir_u);
 }
 
+/* _pier_on_scry_done(): scry callback.
+*/
+static void
+_pier_on_scry_done(void* vod_p, u3_noun nun)
+{
+  u3_pier* pir_u = vod_p;
+  u3_weak res = u3r_at(7, nun);
+
+  if (u3_none == res) {
+    u3l_log("pier: scry failed\n");
+  }
+  else {
+    u3l_log("pier: scry succeeded\n");
+
+    c3_c* pac_c = u3_Host.ops_u.puk_c;
+    if (!pac_c) {
+      pac_c = u3_Host.ops_u.pek_c;
+    }
+
+    u3_noun pad;
+    {
+      u3_noun pax = u3do("stab", u3i_string(pac_c));
+      c3_w len_w = u3kb_lent(u3k(pax));
```

Comment:
This could just be `u3qb_lent(pax)`, which is equivalent but doesn't thrash the count.

Reply:
Huh, I could swear I ran into that not existing, but it's there. Maybe I got confused. Will fix!